requirements/default.txt

```diff
@@ -31,6 +32,7 @@ django-mozilla-product-details==0.7.1 --hash=sha256:c50bffb9828a9139156904a11c50
 django-picklefield==0.3.2 --hash=sha256:5489fef164de43725242d56e65e016137d3df0d1a00672bda72d807f5b2b0d99
 django-pipeline==1.5.4 --hash=sha256:a5a909f403d0ea56f211618949ff4e378f16e5a996bde819bd2b5b0e7a9bb8fe
 django-ratelimit==0.6.0 --hash=sha256:f73b53b2c4fd342cbd4b983ffc33fdcf8f8a6f595d548865ea63c839ef38293c
+djangorestframework==3.3.2 --hash=sha256:5634b1ff56581bf0fe4075e86227fc9693c1ca031c7213c9ae942c445c24817b
```

Comment:
Ah. I used `!sort -df` from vim so it must be slightly different. I can put this back if it makes more sense.